packages/cli/openapi-ir-sdk/fern/definition/finalIr.yml

```diff
@@ -71,6 +77,7 @@ types:
     extends:
       - commons.WithDescription
       - commons.WithSource
+      - commons.WithNamespace
```

Comment:
We need to make sure that all `Webhook`, `Endpoint`, `Channel` all have `commons.WithNamespace` across both the parseIr and finalIr (i believe we are missing in a bunch of places right now)

Reply:
I think these are covered but let me know if you see any specific missing!